### PR TITLE
chore(dist): replace profile id & unify the dependencies/plugins

### PR DESCRIPTION
**Pull request by imbajin**

### CODE REVIEW

File: hugegraph-tools/pom.xml [hunk 1]
```diff
@@ -297,8 +276,8 @@
                     <plugin>
                         <artifactId>maven-release-plugin</artifactId>
                         <configuration>
-                            <!--  Pass these arguments to the deploy plugin.  -->
-                            <arguments>-Prelease</arguments>
+                            <!--  Pass these arguments to the deployment plugin.  -->
+                            <arguments>-Papache-release</arguments>
```
Comment (javeme) [inline hugegraph-tools/pom.xml:280]: "Papache"?

File: hugegraph-tools/pom.xml [hunk 2]
```diff
@@ -241,31 +235,16 @@
...
-                        </fileset>
-                    </filesets>
-                </configuration>
             </plugin>
         </plugins>
     </build>
 
     <profiles>
-        <!-- mvn clean install -Prelease -->
+        <!-- mvn clean install -Papache-release -->
```
Comment (javeme) [inline hugegraph-tools/pom.xml:245]: ditto

File: hugegraph-tools/pom.xml (hunk 1 above)
Comment (imbajin) [reply hugegraph-tools/pom.xml:280]: the profile `id` refers & keep the same with the parent `apache` pom:
https://github.com/apache/maven-apache-parent/blob/99592e229c20f079c1998875e5c24556d217b491/pom.xml#L378

File: pom.xml [hunk 3]
```diff
@@ -481,13 +350,163 @@
...
+                        <encoding>${project.build.sourceEncoding}</encoding>
+                        <compilerArguments>
+                            <Xmaxerrs>500</Xmaxerrs>
+                        </compilerArguments>
+                        <compilerArgs>
+                            <arg>-Xlint:unchecked</arg>
+                        </compilerArgs>
+                    </configuration>
+                </plugin>
+                <!--            <plugin>-->
```
Comment (javeme) [inline pom.xml:371]: useless code? add some comments if keep this code

**Review by javeme — DISMISSED** [review]

File: pom.xml (hunk 3 above)
Comment (imbajin) [reply pom.xml:371]: handle them later (not test well now)

Comment (imbajin) [reply pom.xml:371]: > useless code? add some comments if keep this code

handle them later (not test well now)

**Review by javeme — APPROVED** [review]

**Review by javeme — DISMISSED** [review]

**Review by javeme — DISMISSED** [review]

**Comment by imbajin** [comment]
@javeme this PR will merge until the release done, then use `rebase & merge` option

**Comment by javeme** [comment]
> @javeme this PR will merge until the release done, then use `rebase & merge` option

get it

**Review by javeme — DISMISSED** [review]

**Review by zyxxoo — APPROVED** [review]

**Review by zyxxoo — DISMISSED** [review]

**Review by zyxxoo — APPROVED** [review]

**Pull request merged**